packages/graphql-yoga/src/plugins/resultProcessor/stringify.ts

```diff
@@ -1,9 +1,11 @@
 import { ExecutionResult } from 'graphql'
 import { ExecutionPatchResult } from '../../types'
 
+type MaybeArray<T> = T | T[]
```

Comment:
We should move this to a shared utility file and then use it in other places

Reply:
Makes sense! Done!